docs/standard/serverless-architecture/architecture-approaches/architecture-patterns.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
I talked to Khairun (our SEO expert) about the path. She suggested that we don't create subfolders for the e-book and remove repeated words. So, the final path for this could be something like https://docs.microsoft.com/en-us/dotnet/standard/serverless-architecture/patterns. You can still have the same TOC, but with all files in one directory.

(I know that other e-books might look different but they didn't follow best practices)